azure-service-bus-management/1.0/modules/ROOT/pages/azure-service-bus-management-connector-examples.adoc

```diff
@@ -0,0 +1,248 @@
+= Azure Service Bus Management 1.0 Examples - Mule 4
+
+
+
+. Declare and complete the variables previously described in the corresponding YAML file (for example: `src/main/resources/properties/kafka-producer.yaml`), and load that properties file as a global element.
```

Comment:
Where are the variables previously described? Does this refer to a different doc or section?